torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9064,7 +9064,7 @@ def __call__(self, opinfo, device, dtype, requires_grad, **kwargs):
 foreach_binary_op_db: List[OpInfo] = [
     ForeachFuncInfo(
         "add",
-        foreach_inputs_sample_func(2, True, True),
+        foreach_inputs_sample_func(2, True, True, True),
```

Comment:
Did you verify that the testing code is properly generating the mixed Tensor + alpha examples here which used not to exist?

Reply:
Yes, printing some SampleInputs reveals that alpha is 3, 3.14 or something complex when running test_parity__foreach_add:
```
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'True'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'True'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.........................................................................SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'True'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'True'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'True'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'True'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '(3+3j)', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3.14', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.SampleInput(input=TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]], args=(TensorList[Tensor[20, 20], Tensor[19, 19], Tensor[18, 18], Tensor[17, 17], Tensor[16, 16], Tensor[15, 15], Tensor[14, 14], Tensor[13, 13], Tensor[12, 12], Tensor[11, 11], Tensor[10, 10], Tensor[9, 9], Tensor[8, 8], Tensor[7, 7], Tensor[6, 6], Tensor[5, 5], Tensor[4, 4], Tensor[3, 3], Tensor[2, 2], Tensor[1, 1]]), kwargs={'alpha': '3', 'disable_fastpath': 'False'}, output_process_fn_grad=<function SampleInput.__init__.<locals>.<lambda> at 0x7f912edaf0a0>, broadcasts_input=False, name='')
.................................................................................
```